.PyCharmRunConfs/pytest units.run.xml

```diff
@@ -1,18 +1,3 @@
-<!--
```

Comment:
This file should not have changed, especially since I merged the master branch into takeoff_segment before the PR. Maybe some leftover from the aborted rebase. I changed it back to how it was in be7c82db6b840f14ea9e1ac638155ff2cb0562c6